servers/physics_server_3d.h

```diff
@@ -590,8 +590,8 @@ class PhysicsServer3D : public Object {
 	virtual void joint_set_solver_priority(RID p_joint, int p_priority) = 0;
 	virtual int joint_get_solver_priority(RID p_joint) const = 0;
 
-	virtual void joint_disable_collisions_between_bodies(RID p_joint, const bool p_disable) = 0;
-	virtual bool joint_is_disabled_collisions_between_bodies(RID p_joint) const = 0;
+	virtual void joint_enable_collisions_between_bodies(RID p_joint, const bool p_enable = true) = 0;
```

Comment:
There's no need for the default value (same for 2D and 3D), it's always called with an explicit argument (which is clearer anyway).